### PR TITLE
chore(vpc): fix tests conflict

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Fixes VPC cloud name conflict in tests.

### CODE REVIEW

File: tools/selproj/client.go [hunk 1]
```diff
@@ -9,69 +9,37 @@ import (
...
+func (a *aivenClientAdapter) Projects(ctx context.Context) ([]*aiven.Project, error) {
+	return a.client.Projects.List(ctx)
 }
 
-// ServicesHandlerAdapter adapts aiven.ServicesHandler to ServicesHandler.
-// This is needed because the aiven.ServicesHandler does not implement an interface that is mockable.
-type ServicesHandlerAdapter struct {
-	handler *aiven.ServicesHandler
+// Services returns service list
+func (a *aivenClientAdapter) Services(ctx context.Context, projectName string) ([]*aiven.Service, error) {
```
Comment (Serpentiel) [inline tools/selproj/client.go:38]: let's rename this to `ServicesList`

File: tools/selproj/client.go [hunk 2]
```diff
@@ -9,69 +9,37 @@ import (
...
+	return a.client.Services.List(ctx, projectName)
 }
 
-// List returns a list of services.
-func (a *ServicesHandlerAdapter) List(ctx context.Context, projectName string) ([]*aiven.Service, error) {
-	return a.handler.List(ctx, projectName)
+// VPCs returns vpc list
+func (a *aivenClientAdapter) VPCs(ctx context.Context, projectName string) ([]*aiven.VPC, error) {
```
Comment (Serpentiel) [inline tools/selproj/client.go:43]: and this to `VPCsList`

**Review by Serpentiel — CHANGES_REQUESTED** [review]
some small changes, wdyt?

**Comment by byashimov** [comment]
All [green](https://github.com/aiven/terraform-provider-aiven/actions/runs/7500347581)

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**